diff --git a/node_modules/@tamagui/select/types/index.d.ts b/node_modules/@tamagui/select/types/index.d.ts
index 4a09516..5d54e4f 100644
--- a/node_modules/@tamagui/select/types/index.d.ts
+++ b/node_modules/@tamagui/select/types/index.d.ts
@@ -1,6 +1,6 @@
 export * from './Select';
-export type { SelectContentProps, SelectProps, SelectScrollButtonProps, SelectViewportProps, } from './types';
+export type { SelectContentProps, SelectProps, SelectScrollButtonProps, SelectViewportProps, SelectViewportExtraProps } from './types';
 export * from './context';
-export type { SelectItemProps } from './SelectItem';
+export type { SelectItemProps, SelectItemExtraProps } from './SelectItem';
 export type { SelectTriggerProps } from './SelectTrigger';
 //# sourceMappingURL=index.d.ts.map
